config/samples/config.yaml

```diff
@@ -1,26 +1,69 @@
 clusterName: "example-cluster"
```

Comment:
I just realized this is some free form yaml as opposed to a Kubernetes ConfigMap. I figured that this directory would hold actual full Kubernetes resource definitions, just like the GitSync that lives in this directory. Any reason it doesn't?

Also, do we need 2 files (I see you have `testConfig.yaml` as well) or can we just have 1?

Reply:
The config.yaml file will be mounted as a volume because it is parsed through Viper and will be loaded from /etc/numaplane inside the pod. The other config map you mentioned is located in the manager folder, as seen [here](https://github.com/numaproj-labs/numaplane/blob/feat/repoAuth/config/manager/config.yaml).

Reply:
> The testConfig.yaml I have created is specifically for controller tests, as it contains different values for the configuration to keep things isolated. Moreover, due to frequent changes in the gitsync processor , controller tests often fail. To avoid such occurrences, I have created a separate testConfig.yaml. Should I make it one ?

I think maybe if there's a file that's for a particular test, we should colocate it with the test in a `testData` directory.

Reply:
Noted ,not needed as of now as I have merged them.